### PR TITLE
CLI Packaging + CI/CD

### PR DESCRIPTION
**Pull request by smals-mavh**

### CODE REVIEW

File: .github/workflows/rest-guide-validator-release.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      - name: Display structure of downloaded files
+        run: ls -R
+      - name: Set environment variables
+        run: |
+          $env:SEMANTIC_VERSION=$env:GITHUB_REF_NAME.Substring(1)
+          echo "RELEASED_VERSION=$env:SEMANTIC_VERSION" >> $env:GITHUB_ENV
+      - name: Package Windows Installer
+        run: |
+          jlink --add-modules java.base,java.naming,java.xml,java.desktop --output ./target/custom-jre --strip-debug --no-header-files --no-man-pages --verbose
+          jpackage --input ./target --name belgif-validate-openapi --main-jar belgif-rest-guide-validator-cli-$env:RELEASED_VERSION.jar --type msi --app-version $env:RELEASED_VERSION --description "The belgif-rest-guide-validator is used to validate if an OpenAPI document conforms to the guidelines in the Belgif REST guide." --vendor "Belgif" --icon ./package/belgif.ico --win-console --resource-dir "./package/windows" --runtime-image ./target/custom-jre --dest ./target --install-dir belgif-rest-guide-validator --add-launcher right-click-and-validate="./package/right-clickable-launcher.properties" --verbose
```
Comment (pvdbosch) [inline .github/workflows/rest-guide-validator-release.yml:68]: shorten description "Validate OpenAPI to Belgif guidelines"
Rename "right-click-and-validate"; best that it includes belgif validator somewhere and can be used as double-click if set as default. Maybe "explorer-belgif-rest-guide-validator", or a better name if you can think of one...

File: readme.md [hunk 2]
```diff
@@ -230,6 +229,23 @@ You can execute the plugin to validate the api files without breaking the build
 | jsonOutputFile | File | ${rest-guide-validator.outputDir}/validationReport.json | Output file for JSON validation report.  | rest-guide-validator.jsonOutputFile | 
 | groupBy   | rule / file | rule | Specify how you want to group the violation output                                                                          | rest-guide-validator.groupBy |
 
+# Standalone belgif-rest-guide-validator
```
Comment (pvdbosch) [inline readme.md:232]: this section is buried within the Maven documentation. If you start reading the README, you'd still think it's a plugin that requires Maven.
Best to open README with scope of the tool (without naming it plugin), mentioning choice Maven plugin or standalone executable;  then separate chapter per choice.

**Review by pvdbosch — CHANGES_REQUESTED** [review]

**Comment by pvdbosch** [comment]
Could you try out the --win-per-user-install option of jpackage? It might allow installation without admin rights.
```
--win-per-user-install
Request to perform an install on a per-user basis
```

File: readme.md (hunk 2 above)
Comment (smals-mavh) [reply readme.md:232]: Done

**Review by pvdbosch — APPROVED** [review]

**Pull request merged**